packages/xo-server-netbox/src/index.js

```diff
@@ -213,9 +211,15 @@ class Netbox {
     }
   }
 
+  async #fetchNetboxVersion() {
+    // Endpoint supported since v2.10. If not supported, Netbox needs to be updated.
```

Comment:
Is there any documentation that say `netbox` must be at least in `v2.10` to be used with this plugin?

Reply:
Actually, thinking about it, I don't think there's any point in failing if the endpoint isn't supported so I'm changing it to keep trying without the version instead.